data/json/items/gun/chemical_spray.json

```diff
@@ -15,7 +15,8 @@
     "to_hit": -1,
     "bashing": 9,
     "material": [ "steel", "plastic" ],
-    "flags": [ "FIRE_50", "NEVER_JAMS", "NON_FOULING" ],
+    "flags": [ "NEVER_JAMS", "NON_FOULING" ],
```

Comment:
If self-made/makeshift, should it be non-fouling?

Reply:
It is a laser gun, it doesn't emit anything to clog up the lenses.

Reply:
I thought this was a chemical spray?